docs/iris/src/whatsnew/latest.rst

```diff
@@ -0,0 +1,57 @@
+.. include:: ../common_links.inc
+
+<unreleased>
```

Comment:
As you suggested, your "What's New" entries should now be part of [`3.0.rst`](https://github.com/SciTools/iris/blob/6d07d8dc1663886606cbf611ec0d810bb3daf92b/docs/iris/src/whatsnew/3.0.rst), and `latest.rst` should therefore not exist.

(This can be inferred from the fact that [separate "What's New" pages](https://scitools-iris.readthedocs.io/en/latest/whatsnew/index.html) only exist for major and minor releases, but not bugfix releases or release candidates).

Reply:
> (This can be inferred from the fact that [separate "What's New" pages](https://scitools-iris.readthedocs.io/en/latest/whatsnew/index.html) only exist for major and minor releases, but not bugfix releases or release candidates).

Ah ok nice

Reply:
Hopefully the entries are now in the right order, can switch around if needed (or I think iris admins should be able to push to this branch too if you don't want to wait for me)